packages/mobile/src/screens/tip-artist-screen/SendTipScreen.tsx

```diff
@@ -1,20 +1,46 @@
-import { useCallback } from 'react'
+import { useCallback, useState } from 'react'
 
+import { StringAudio } from 'audius-client/src/common/models/Wallet'
+import { getAccountBalance } from 'audius-client/src/common/store/wallet/selectors'
+import { parseAudioInputToWei } from 'audius-client/src/common/utils/wallet'
+
+import IconArrow from 'app/assets/images/iconArrow.svg'
 import IconRemove from 'app/assets/images/iconRemove.svg'
-import { Button, Screen } from 'app/components/core'
+import { Button } from 'app/components/core'
 import { useNavigation } from 'app/hooks/useNavigation'
+import { useSelectorWeb } from 'app/hooks/useSelectorWeb'
+import { makeStyles } from 'app/styles'
 
 import { TopBarIconButton } from '../app-screen'
 
+import { AvailableAudio } from './AvailableAudio'
+import { ErrorText } from './ErrorText'
+import { ReceiverDetails } from './ReceiverDetails'
+import { TipInput } from './TipInput'
+import { TipScreen } from './TipScreen'
 import { TipArtistNavigationParamList } from './navigation'
 
 const messages = {
-  sendTip: 'Send Tip'
+  sendTip: 'Send Tip',
+  insufficientBalance: 'Insufficient Balance'
 }
 
+const useStyles = makeStyles(({ spacing }) => ({
+  sendButton: {
+    marginBottom: spacing(6)
+  }
+}))
+
 export const SendTipScreen = () => {
+  const styles = useStyles()
+  const [tipAmount, setTipAmount] = useState('')
+  const accountBalance = useSelectorWeb(getAccountBalance)
   const navigation = useNavigation<TipArtistNavigationParamList>()
 
+  const hasInsufficientBalance =
+    accountBalance &&
+    parseAudioInputToWei(tipAmount as StringAudio)?.gte(accountBalance)
```

Comment:
should it just be greater than, aka `gt` instead of `gte`?

Reply:
also is there a case where we have no accountBalance? I'm guessing not, but maybe `!accountBalance || parse...` instead?